src/main/kotlin/no/nav/helse/spenn/simulering/SimuleringService.kt

```diff
@@ -28,6 +31,23 @@ class SimuleringService(val simulerFpService: SimulerFpService) {
         private val log = LoggerFactory.getLogger(SimuleringService::class.java)
     }
 
+    fun runSimulering(oppdrag: OppdragStateDTO): OppdragStateDTO {
+        log.info("simulering for ${oppdrag.soknadId}")
+        val result = callSimulering(oppdrag)
+        val status = when (result.status) {
+            Status.OK -> OppdragStateStatus.SIMULERING_OK
+            else -> OppdragStateStatus.SIMULERING_FEIL
+        }
+        return oppdrag.copy(simuleringResult = result, status = status)
+    }
+
+    private fun callSimulering(oppdrag: OppdragStateDTO): SimuleringResult {
```

Comment:
jeg er fan av ternaries for enkle caser som dette ;) da kan du bruke implisitt return også <3